lib/languages/raku.txt

```diff
@@ -0,0 +1,11 @@
+!e
+#`(+\j-->>)+\pj
+#`{+\j-->>}+\pj
+#`[+\j-->>]+\pj
+#`<+\j-->>>+\pj
+#`(\j-->>)\pj
+#`{\j-->>}\pj
+#`[\j-->>]\pj
+#`<\j-->>>\pj
+#
+=begin-->>=end
```

Comment:
I would suggest that this be `=begin comment-->>=end comment` with an additional `=comment`, unless there is some additional Pod you wish to cover?

https://docs.raku.org/language/pod#Pod6_comments

Reply:
I've added these - thank you. :)

And another thank you for [fixing the documentation around ｢｣](https://github.com/Raku/doc/issues/4110). I've added support for that too.